backend/store/issue.go

```diff
@@ -454,6 +456,10 @@ func (s *Store) ListIssueV2(ctx context.Context, find *FindIssueMessage) ([]*Iss
 	if v := find.Offset; v != nil {
 		limitOffsetClause += fmt.Sprintf(" OFFSET %d", *v)
 	}
+	if len(find.LabelList) != 0 {
+		where = append(where, fmt.Sprintf("payload->'labels' ?& ARRAY[$%d]", len(args)+1))
+		args = append(args, strings.Join(find.LabelList, ","))
```

Comment:
```suggestion
		where = append(where, fmt.Sprintf("payload->'labels' ?& $%d::TEXT[]", len(args)+1))
		args = append(args, find.LabelList)
```
This might just work.

Reply:
TIL

Reply:
Squel boy